### PR TITLE
Fixes #35462 - Depend on the rss gem

### PR DESCRIPTION
**Pull request by ekohl**
Ruby 3.0 has moved rss from a default gem to a bundled gem. This means it must be specified as a dependency. See https://community.theforeman.org/t/path-to-ruby-3-0-3-1-el9-and-ubuntu-22-04/30295#ruby-30-4 for more details.

### CODE REVIEW

File: Gemfile
Comment (ekohl) [inline Gemfile:53]: Question is if we should make this conditional or also pull it in on Ruby 2.7.

In this current form the dependency generator for RPM will use the Ruby version that runs it (which for me is Ruby 3.1 since I'm on Fedora, but could be different for others). On the other hand, Debian packaging will use the real Ruby.

Comment (ares) [reply Gemfile:53]: it would be nice to avoid the if condition here IMHO, but not a big deal if that's not possible, it's just something we'd need to drop eventually when we support Ruby 3+

**Comment by stejskalleos** [comment]
[test katello]

File: Gemfile
Comment (stejskalleos) [reply Gemfile:53]: > it would be nice to avoid the if condition here IMHO

Just curious, how else would you do it?

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**, tested on `3.0.4`, works as expected.

Thanks @ekohl !

**Comment by stejskalleos** [comment]
Waiting for approval from the @theforeman/packaging team

**Review by evgeni — APPROVED** [review]
THE PACKAGING TEAM APPROVES!

**Pull request merged**
